.github/workflows/update-private-ca-lambda.yml

```
@@ -0,0 +1,33 @@
+name: Update Private CA Lambda on push
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - private-ca/server/
```

Add ** to include sub folders
```suggestion
      - private-ca/server/**
```

---

Done